### ISSUE TITLE
More compact design (more text on screen at once)

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#67]
> perhaps smaller or different fonts option and smaller icons (I suppose I could just zoom out the page but that doesn't do a clean job)

Requested by LateNightHacks via user review 2017/10/14.

**Comment by ShervinM** [comment]
not really sure what makes the TF window's content look so much bigger to me when I put it side by side TO. it seems to be either the font itself of the white colour for open tabs vs. blue on TO. 
When I do %90 zoom everything looks the right density, even then though TO icons still look smaller, or am I imagining it... (ok, took a screenshot of both windows next to each other then cut icons from one window and put it next to the matching one on the other, at %90 zoom on TF, favicons are still one pixel larger than TO at %100 zoom. fonts are both 8pixel high)

**Comment by cxw42** [comment]
@ShervinM would you please post the side-by-side screenshot (without any tabsets having confidential information)?  I'd like to take a look.  In 0226585dc6e45f344b30a9c2b419f99d443a8123 ([diff](https://github.com/cxw42/TabFern/commit/0226585dc6e45f344b30a9c2b419f99d443a8123#diff-4766606ec3b570e03a2b4c59d54cda6f)) I actually opened up the line spacing by 1px because I found that easier to read.  It would not be hard to make that optional.

**Comment by ShervinM** [comment]
here you go, side by side, TF@%100, TF@%90 and TO@%100
I made the yellow boxes at pixel level and they represent equal size, so you can count the dots and compare, same for the icons, look at how they fit inside their square and the gaps.
![d-11](https://user-images.githubusercontent.com/7252356/33155447-ea10b092-d044-11e7-80b6-7ffcc071f2af.jpg)
![d-12](https://user-images.githubusercontent.com/7252356/33155457-f9aceb9c-d044-11e7-88db-b9b81a475286.jpg)

**Comment by ShervinM** [comment]
the icons in TF actually look a bit darker and fuzzier as well, which just started to make sense to me.
favicons are standard 16x16 pixels, TO shows them as 16x16 so they look as they should be. on TF they are shown as 18x18 pixels which is nonstandard, hence the browser resizes them to fit, that would also cause a re-rendering which is the reason behind the duller colours.

**Comment by ShervinM** [comment]
Any ideas on this one? did I make any sense?

**Comment by cxw42** [comment]
Yes, it makes sense, and I greatly appreciate the good data!  I agree with your conclusions two comments ago.  Open/close of individual tabs is higher on my list, but I will work on this when I can.  I do think this one is all CSS.
